CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ sudo dnf install make automake gcc gcc-c++ kernel-devel
 You can run the tests of the project that you modified by going to the project's directory and running:
 
 ```shell
-pnpm test
+pnpm test-main
```

Comment:
This should be reverted.

Reply:
There is no npm command called 'test' in root project

Reply:
It is not about the root project.

The text says "by going to the project's directory"